pkg/engine/engine.go

```diff
@@ -181,28 +181,20 @@ func getStorageAccountType(sizeName string) (string, error) {
 }
 
 func makeMasterExtensionScriptCommands(cs *api.ContainerService) string {
-	copyIndex := "',copyIndex(),'"
-	if cs.Properties.OrchestratorProfile.IsKubernetes() {
-		copyIndex = "',copyIndex(variables('masterOffset')),'"
-	}
 	return makeExtensionScriptCommands(cs.Properties.MasterProfile.PreprovisionExtension,
-		cs.Properties.ExtensionProfiles, copyIndex)
+		cs.Properties.ExtensionProfiles)
 }
 
 func makeAgentExtensionScriptCommands(cs *api.ContainerService, profile *api.AgentPoolProfile) string {
-	copyIndex := "',copyIndex(),'"
-	if profile.IsAvailabilitySets() {
-		copyIndex = fmt.Sprintf("',copyIndex(variables('%sOffset')),'", profile.Name)
-	}
 	if profile.OSType == api.Windows {
 		return makeWindowsExtensionScriptCommands(profile.PreprovisionExtension,
-			cs.Properties.ExtensionProfiles, copyIndex)
+			cs.Properties.ExtensionProfiles)
 	}
 	return makeExtensionScriptCommands(profile.PreprovisionExtension,
-		cs.Properties.ExtensionProfiles, copyIndex)
+		cs.Properties.ExtensionProfiles)
 }
 
-func makeExtensionScriptCommands(extension *api.Extension, extensionProfiles []*api.ExtensionProfile, copyIndex string) string {
+func makeExtensionScriptCommands(extension *api.Extension, extensionProfiles []*api.ExtensionProfile) string {
```

Comment:
This *feels* like a bug, but `copyIndex` appears to be unused in these functions.